src/sonic-host-services/scripts/caclmgrd

```diff
@@ -273,6 +274,41 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
         return allow_internal_docker_ip_cmds
 
+    def get_loopback_ip_from_loopback_table(self, loopback_table):
```

Comment:
please provide "loopback3" as an argument so that this function can be re-used later